documentation/src/docs/segment-store-service.md

```diff
@@ -66,117 +96,116 @@ Segment Containers are a logical grouping of Segments, and are responsible for a
 - **Tier 1 Metadata**: Various pieces of information that can be used to accurately truncate the Tier 1 Storage Log once all operations prior to that point have been durably stored to Tier 2.
-- **Checkpoints**: Container Metadata is periodically checkpointed by having its entire snapshot (including Active Segments) serialized to Tier 1. A Checkpoint serves as a Truncation Point for Tier 1, meaning it contains all the updates that have been made to the Container via all the processed operations before it, so we no longer need those operations in order to reconstruct the Metadata. If we truncate Tier 1 on a Checkpoint, then we can use information from Tier 2 and this Checkpoint to reconstruct what is in the Metadata previously, without relying on any operation prior to it in Tier 1.
+- **Checkpoints**: Container Metadata is periodically checkpointed by having its entire snapshot (including Active Segments) serialized to Tier 1. A Checkpoint serves as a Truncation Point for Tier 1, as it contains all the updates that have been made to the Container via all the processed operations before it, so we no longer need those operations in order to reconstruct the metadata. If we truncate Tier 1 on a Checkpoint, then we can use information from Tier 2 and this Checkpoint to reconstruct by using the previously available metadata, without relying on any operation prior to it in Tier 1.
 
 #### Segment Metadata
 Each Segment Container needs to keep per-segment metadata, which it uses to keep track of the state of each segment as it processes operations for it. The metadata can be volatile (it can be fully rebuilt upon recovery) and contains the following properties for each segment that is currently in use:
 
-- `Name`: The name of the Segment.
-- `Id`: Internally assigned unique Segment Id. This is used to refer to Segments, which is preferred to the Name. This Id is sticky for the lifetime of the Segment, which means that even if the Segment becomes inactive, a future reactivation will have it mapped to the same Id.
-- `StartOffset` (also known as `TruncationOffset`): The lowest offset of the data that is available for reading. A non-truncated segment will have Start Offset equal to 0, while subsequent Truncate operations will increase (but never decrease) this number.
+- `Name`: The name of the Stream Segment.
+- `Id`: Internally assigned unique Stream Segment ID. This is used to refer to Stream Segments, which is preferred to the Name. This ID is used for the entire lifetime of the Stream Segment, which means that even if the Stream Segment becomes inactive, a future reactivation will have it mapped to the same ID.
+- `StartOffset` (also known as `TruncationOffset`): The lowest offset of the data that is available for reading. A non-truncated Stream Segment will have Start Offset equal to _0_, while subsequent Truncate Operations will increase (but never decreases) this number.
```

Comment:
"decreases" -> "decrease" . Leave it as it was. This is in conjunction with a plural subject, hence we need the plural form.

Reply:
Accepted suggestion